myriad-scheduler/build.gradle

```diff
@@ -69,3 +74,36 @@ sourceSets {
         }
     }
 }
+
+node {
+    // Version of node to use.
+    version = '4.2.1'
+
+    // Base URL for fetching node distributions (change if you have a mirror).
+    distBaseUrl = 'https://nodejs.org/dist'
+
+    // If true, it will download node using above parameters.
+    // If false, it will try to use globally installed node.
+    download = true
+
+    // Set the work directory where node_modules should be located
+    nodeModulesDir = file("${project.projectDir}/src/main/resources/webapp")
+}
+
+gulp {
+    // Set the directory where gulpfile.js should be found
+    workDir = file("${project.projectDir}/src/main/resources/webapp")
+
+    // Whether colors should output on the terminal
+    colors = false
```

Comment:
ha! sometimes colors just print weird control characters if your terminal can't handle it. you're not missing anything too exciting.